business/workloads.go

```diff
@@ -95,15 +100,15 @@ func (in *WorkloadService) GetWorkloadList(namespace string, linkIstioResources
 	go func() {
 		defer wg.Done()
 		var err2 error
-		ws, err2 = fetchWorkloads(in.businessLayer, namespace, "")
+		ws, err2 = fetchWorkloads(in.businessLayer, criteria.Namespace, "")
```

Comment:
Totally +1.
I'm thinking in consolidate the Services part first, just to avoid a large PR, but I won't close the issue and the same strategy should go in the same issue.

The `fetchWorkloads` was an aux method but it has grown and it deserves the same design.